src/QandaAtom.tsx

```diff
@@ -1,7 +1,445 @@
 import React from 'react';
-
 import { QandaAtomType } from './types';
+import { css } from 'emotion';
+import { neutral, error } from '@guardian/src-foundations/palette';
+import { headline, textSans, body } from '@guardian/src-foundations/typography';
+
+const { useState } = React;
+
+// CSS
+const detailStyling = css`
+    margin: 16px 0 36px;
+    background: ${neutral[93]};
+    padding: 0 5px 6px;
+    border-image: repeating-linear-gradient(
+            to bottom,
+            ${neutral[86]},
+            ${neutral[86]} 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+    border-top: 13px solid black;
+    border-left: 0;
+    border-right: 0;
+    border-bottom: 0;
+    position: relative;
+    summary {
+        list-style: none;
+        margin: 0 0 16px;
+    }
+    summary::-webkit-details-marker {
+        display: none;
+    }
+
+    @media (min-width: 46.25em) {
+        ul {
+            list-style: none;
+            margin-bottom: 1rem;
+        }
+    }
+
+    ul {
+        list-style: none;
+        margin: 0 0 0.75rem;
+        padding: 0;
+    }
+
+    li {
+        margin-bottom: 0.375rem;
+        padding-left: 1.25rem;
+    }
+
+    li:before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.375rem;
+        height: 0.75rem;
+        width: 0.75rem;
+        margin-right: 0.5rem;
+        background-color: ${neutral[86]};
+        margin-left: -1.25rem;
+    }
+
+    a {
+        color: #ab0613;
+        text-decoration: none !important;
+        border-bottom: 0.0625rem solid #bdbdbd;
+        transition: border-color 0.15s ease-out;
+    }
+
+    a:hover {
+        border-bottom: solid 0.0625rem ${error[400]};
+    }
+
+    b {
+        font-weight: bold;
+    }
+
+    summary:focus {
+        outline: none;
+    }
+`;
+
+const figureStyling = css`
+    display: block;
+    position: relative;
+    margin-bottom: 0.75rem;
+    margin-top: 1rem;
+
+    details[open] .is-on {
```

Comment:
Ah ok, here is the `is-on`, `is-off` - I would do the conditonal classes with js teneary operator here as you have the `expanded` state anyway (it's quite nice using the css classes, but it will be easier for future maintainers to reason with if we use one way to define and read state)

So below instead of `<span className={'is-on ' + iconSpacing}>` use `hasBeenExpanded && <span className={iconSpacing}>`